src/AdminConsole/Components/Shared/Credentials.razor.cs

```diff
@@ -46,11 +42,18 @@ public IReadOnlyCollection<CredentialModel> GetItems()
     [Parameter]
     public bool HideDetails { get; set; }
 
+    [Parameter]
+    public required IPasswordlessClient PasswordlessClient { get; set; }
```

Comment:
question: Why doesn't it work with the injected one, why do we need to pass this as a parameter?

Reply:
This component is used on 3 pages in total:

- /account/useronboarding: uses `IPasswordlessClient`
- /account/profile: uses `IPasswordlessClient`
- /app/<appid>/credentials/list: uses `IScopedPasswordlessClient` (which replaces the ApiSecret from admin app with the one from the customer's app)